### DISCUSSION TITLE
unable to validate game version

### DISCUSSION THREAD

**Discussion by Th3AnG3L** [azerothcore/azerothcore-wotlk#14391] [Guides & Tips]
Hello i follow the steps how to create the server
but when i open client wow.exe and try connect it shows this error "wow unable to validate game version"
on bottom screen it shows version 12341
on db i see it shows 12340

how i can solve the issue?

**Comment by Kitzunu** [comment]
Get a correct client. 12340 build is for 3.3.5a. 12431 is something custom

**Comment by mrcool72** [comment]
I have been running this server for quite some time and the server restarted from updates, then I added another realm and now I am getting Unable to Validate Game Version.
Game Version says in the bottom corner: 12340
Database says same number in the realmlist table under gamebuild.
in the auth table, it is now showing another row: 13930 3.3.5a
Is there someplace else that I could be missing something, that I should look at?
Do I just update my client to the new build now?  Delete that row in the DB?

**Comment by TheSCREWEDSoftware** [comment]
`acore_auth.build_info` if you wish to allow other clients to be see the realm, keep in mind if the client has different opcodes (as the example query below, which is for Cata 4.3.4 (15595), it wont work)

```sql
DELETE FROM `build_info` WHERE `build`=15595;
INSERT INTO `build_info` (`build`, `majorVersion`, `minorVersion`, `bugfixVersion`, `hotfixVersion`, `winAuthSeed`, `win64AuthSeed`, `mac64AuthSeed`, `winChecksumSeed`, `macChecksumSeed`) VALUES 
(15595, 4, 3, 4, NULL, NULL, NULL, NULL, NULL, NULL);
```
